contracts/evoting/evoting.go

```diff
@@ -32,7 +32,7 @@ import (
 const (
 	shufflingProtocolName = "PairShuffle"
 	errGetTransaction     = "failed to get transaction: %v"
-	errGetElection        = "failed to get election: %v"
+	errGetForm        = "failed to get form: %v"
```

Comment:
it seems that your IDE is not formatting your go code for you. Please make sure it does by installing the recommended tooling for go code.